packages/expect/src/matchers.js

```diff
@@ -87,106 +85,66 @@ const matchers: MatchersObject = {
     const secondArgument = arguments.length === 3 ? 'precision' : null;
     ensureNumbers(actual, expected, '.toBeCloseTo');
     const pass = Math.abs(expected - actual) < Math.pow(10, -precision) / 2;
-    const message = pass
-      ? () =>
-          matcherHint('.not.toBeCloseTo', undefined, undefined, {
-            secondArgument,
-          }) +
-          '\n\n' +
-          `Expected value not to be close to (with ${printExpected(
-            precision,
-          )}-digit precision):\n` +
-          `  ${printExpected(expected)}\n` +
-          `Received:\n` +
-          `  ${printReceived(actual)}`
-      : () =>
-          matcherHint('.toBeCloseTo', undefined, undefined, {
-            secondArgument,
-          }) +
-          '\n\n' +
-          `Expected value to be close to (with ${printExpected(
-            precision,
-          )}-digit precision):\n` +
-          `  ${printExpected(expected)}\n` +
-          `Received:\n` +
-          `  ${printReceived(actual)}`;
+    const message = () =>
+      matcherHint('.toBeCloseTo', undefined, undefined, {
+        isNot: this.isNot,
+        secondArgument,
+      }) +
+      '\n\n' +
+      `Precision: ${printExpected(precision)}-digit\n` +
```

Comment:
I'd move precision below received

Reply:
See pictures for an alternative where to move precision.

Reply:
Love it